src/misc/drowning.ts

```diff
@@ -0,0 +1,89 @@
+import { Bukkit, GameMode } from 'org.bukkit';
+import { Player } from 'org.bukkit.entity';
+import { ItemStack } from 'org.bukkit.inventory';
+import { minMax } from '../common/helpers/math';
+
+// Min weight to start drowning (with minimum force)
```

Comment:
When writing comments for variables like here, use JSDoc so that they also get editor tooltips.